snapcraft/plugins/python.py

```diff
@@ -53,40 +53,25 @@
 """
 
 import collections
-import json
+import contextlib
 import os
 import re
-import shutil
 import stat
-import subprocess
-import tempfile
-from contextlib import contextmanager, suppress
-from glob import glob
+from contextlib import contextmanager
```

Comment:
you already `import contextlib` above

Reply:
Well what do you know! Fixed, thank you.